src/app/zap-templates/zcl/data-model/chip/ethernet-network-diagnostics-cluster.xml

```diff
@@ -35,14 +35,14 @@ limitations under the License.
     <code>0x0037</code>
     <define>ETHERNET_NETWORK_DIAGNOSTICS_CLUSTER</define>
     <description>The Ethernet Network Diagnostics Cluster provides a means to acquire standardized diagnostics metrics that MAY be used by a Node to assist a user or Administrative Node in diagnosing potential problems.</description>
-    <attribute side="server" code="0x00" define="PHY_RATE" type="ENUM8" writable="false" optional="true">PHYRate</attribute>
-    <attribute side="server" code="0x01" define="FULL_DUPLEX" type="BOOLEAN" min="0x00" max="0x01" writable="false" default="0x00" optional="true">FullDuplex</attribute>
+    <attribute side="server" code="0x00" define="PHY_RATE" type="ENUM8" writable="false" isNullable="true" optional="true">PHYRate</attribute>
```

Comment:
```suggestion
    <attribute side="server" code="0x00" define="PHY_RATE" type="PHYRateType" writable="false" isNullable="true" optional="true">PHYRate</attribute>
```

Reply:
Hit the following compile error after applying this change.

```
../../../examples/lighting-app/linux/third_party/connectedhomeip/src/lib/core/Optional.h: In instantiation of ‘constexpr T& chip::Optional<T>::Emplace(Args&& ...) [with Args = {unsigned char&}; T = EmberAfPHYRateType]’:
../../../examples/lighting-app/linux/third_party/connectedhomeip/src/app/data-model/Nullable.h:56:36:   required from ‘constexpr T& chip::app::DataModel::Nullable<T>::SetNonNull(Args&& ...) [with Args = {unsigned char&}; T = EmberAfPHYRateType]’
../../../examples/lighting-app/linux/third_party/connectedhomeip/src/app/clusters/ethernet-network-diagnostics-server/ethernet-network-diagnostics-server.cpp:81:34:   required from here
../../../examples/lighting-app/linux/third_party/connectedhomeip/src/lib/core/Optional.h:124:9: error: invalid conversion from ‘unsigned char’ to ‘EmberAfPHYRateType’ [-fpermissive]
  124 |         new (&mValue.mData) T(std::forward<Args>(args)...);
      |         ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
      |         |
      |         unsigned char
cc1plus: error: unrecognized command line option ‘-Wno-unknown-warning-option’ [-Werror]
```

Reply:
solved by pHYRate.SetNonNull(static_cast<PHYRateType>(value));